### PR TITLE
add geckodriver support

### LINKED ISSUE #17: Support geckodriver
At the moment, the user can only use the app if they have chromedriver, so I propose to create a flag, `firefox` or `geckodriver`, to enable the use of geckodriver instead of chromedriver

### PR DESCRIPTION
**Pull request by HaruNashii**
this closes #17

### CODE REVIEW

File: src/player/watch_media.rs [hunk 1]
```diff
@@ -11,25 +10,43 @@ use crate::{
...
 #[tokio::main]
 pub async fn watch_media(media: Media, img_mode: Option<bool>) -> WebDriverResult<()> {
     let language = TRANSLATION.get().unwrap();
     let use_mpv = USE_MPV.get().unwrap();
+    let use_geckodriver = USE_GECKODRIVER.get().unwrap();
 
     let url = format!("https://vizer.in/{}", &media.url);
-    let mut chromedriver = Command::new("chromedriver").spawn().unwrap();
-    // we need to wait chromedriver to start :(
+    let mut command_name = String::new();
```
Comment (anotherlusitano) [inline src/player/watch_media.rs:23]: Change the name to driver_command

File: src/player/watch_media.rs [hunk 2]
```diff
@@ -11,25 +10,43 @@ use crate::{
...
+    if *use_geckodriver {
+        command_name.push_str("geckodriver");
+    } else {
+        command_name.push_str("chromedriver");
+    };
+
+    let mut command = Command::new(command_name)
```
Comment (anotherlusitano) [inline src/player/watch_media.rs:30]: Instead of `command` name it `browser_driver`

**Review by anotherlusitano — CHANGES_REQUESTED** [review]
Its all good you just need to change some variables names :)

File: src/player/watch_media.rs (hunk 2 above)
Comment (HaruNashii) [reply src/player/watch_media.rs]: Done :)

File: src/player/watch_media.rs (hunk 1 above)
Comment (HaruNashii) [reply src/player/watch_media.rs]: Okaayy, :) done.

**Review by anotherlusitano — APPROVED** [review]
:rocket:

**Pull request merged**